src/components/ContactInfo/index.jsx

```diff
@@ -1,103 +1,192 @@
+          index
+        ) => {
+          const isLast = index === contactData.length - 1;
+          return (
+            <div
+              key={_id}
+              className={cn(
+                'contact',
+                'my-1',
+                'lg:my-0',
+                'w-64',
+                'mt-2',
+                'md:mt-0',
+                {
+                  'border-right': !isLast
+                }
+              )}
+            >
+              <div className="contact-icon">
+                <img className="h-auto" src={iconUrl} alt={name} />
+              </div>
+              <div className="information">
                 <a
-                  className="full-width-click"
+                  className="block"
                   href={url}
                   target="_blank"
                   rel="noopener noreferrer"
                 >
-                  <i className={fontawesome_icons}></i>
+                  {title}
                 </a>
+                <small className="information-small leading-normal">
+                  {sub_title}
+                </small>
               </div>
-            ))}
-          </div>
-
-          <div className="contact-section__contacts flex flex-no-wrap flex-col lg:flex-row justify-end text-sm mx-6">
-            {contactData.map(({ _id, title, url, sub_title, icon }, index) => {
-              const isLast = index === contactData.length - 1;
-              return (
-                <div
-                  key={_id}
-                  className={cn(
-                    'contact',
-                    'my-1',
-                    'lg:my-0',
-                    'w-64',
-                    'mt-2',
-                    'md:mt-0',
-                    {
-                      'border-right': !isLast
-                    }
-                  )}
-                >
-                  <div className="contact-icon">
-                    <img className="h-auto" src={icon.url} alt={icon.name} />
-                  </div>
-                  <div className="information">
-                    <a
-                      className="block"
-                      href={url}
-                      target="_blank"
-                      rel="noopener noreferrer"
-                    >
-                      {title}
-                    </a>
-                    <small className="information-small leading-normal">
-                      {sub_title}
-                    </small>
-                  </div>
-                </div>
-              );
-            })}
-          </div>
-        </div>
-      </div>
-    </section>
+            </div>
+          );
+        }
+      )}
+    </div>
   );
 };
+
+const CircleLoading = () => (
+  <ContentLoader
+    speed={2}
+    width={50}
+    height={50}
+    viewBox="0 0 100 100"
+    backgroundColor="#f5f5f5"
+    foregroundColor="#f5f5f5"
+  >
+    <circle cx="40" cy="40" r="33" />
+  </ContentLoader>
+);
+
+const ContactLoader = () => (
```

Comment:
How about something like this?
```js
const ContactLoader = ({count}) => {
return Array(count).fill(1).map(()=>(<ContentLoader
    speed={2}
    width={200}
    height={60}
    viewBox="0 0 200 60"
    backgroundColor="#f5f5f5"
    foregroundColor="#f5f5f5"
  >
    <circle cx="46" cy="26" r="21" />
    <rect x="74" y="10" rx="0" ry="0" width="163" height="8" />
    <rect x="80" y="30" rx="0" ry="0" width="94" height="8" />
  </ContentLoader>))
}
```

and using it like this
```js
<ContactLoader count=3 />
```

Reply:
I Like the way you mentee us 😃